wordpress-mobile/WordPress-Android
# Enable encrypted logging and add tracking

This PR re-enables encrypted logging and adds tracking. 738a93400395cf0eecf46a7061b189d275334a15 is the revert commit, so it may be skipped during review, so the only code to review is 8a976ae34761dbaf6131cec3bcbe9acd8644f8e6.

**To test:**
* Crash the app using instructions from #12707
* Re-open the app
* Find the crash event in Sentry from https://sentry.io/share/issue/04a76a1c7b6240c699fe4b907668716c/ and grab its uuid
* Use the uuid to verify the logs are successfully uploaded to MC

**PR submission checklist:**
